### PR TITLE
Add initial implementation

### PR DESCRIPTION
**Pull request by RDPerera**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: .connector-store/.connector-store/meta.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Adobe Campaign Classic",
+  "owner": "WSO2",
+  "category": "Developer Tools",
```
Comment (arunans23) [inline .connector-store/.connector-store/meta.json]: Does this come under Developer Tools? Shall we rename it to suitable category? Maybe "Marketing/Social Media"

File: src/main/assembly/filter.properties [hunk 2]
```diff
@@ -0,0 +1,3 @@
+product.name=WSO2 ESB
```
Comment (arunans23) [inline src/main/assembly/filter.properties]: Even though this file is not used, shall we update these files with MI related values?

**Review by arunans23 — DISMISSED** [review]

File: .connector-store/.connector-store/meta.json (hunk 1 above)
Comment (RDPerera) [reply .connector-store/.connector-store/meta.json]: As we discussed offline, the Category changed to `Sales and Marketing`.

Changed with [45a4639](https://github.com/wso2-extensions/esb-connector-adobecampaignclassic/pull/1/commits/45a46394817c0dde53960be37bd29ea97ba52435).

File: src/main/assembly/filter.properties (hunk 2 above)
Comment (RDPerera) [reply src/main/assembly/filter.properties]: Sure changed with [33fafe7](https://github.com/wso2-extensions/esb-connector-adobecampaignclassic/pull/1/commits/33fafe7b3b159683b566791f32a73c0bc28127d3).

**Review by arunans23 — APPROVED** [review]

**Pull request merged**